src/Hive.Core/sia380/sia380.py

```diff
@@ -27,11 +27,16 @@
 HOURS_PER_YEAR = 8760  # no leap year    
 SECONDS_PER_HOUR = 3600.0
 
+CELCIUS_TO_KELVIN = 273.15 # [K]
+GRAVITATIONAL_CONSTANT = 9.8 # [m/s^2]
+RHO = 1.2       # Luftdichte in kg/m^3, average as actually dependent on temperature (and more..?)
+C_P = 1005      # Spez. Wärmekapazität Luft in J/(kgK)
+
 def cleanDictForNaN(d):
     # a = d.values()
     # b = d.keys()
     for i in d:
-        # if isinstance(d[i],str): continue # DEBUG
+        # if isinstance(d[i],str) or isinstance(d[i], unicode): continue # DEBUG
```

Comment:
That's a good way to solve it :) A couple things i won't really be able to do this way (such as importing grasshopper stuff), but it will definitely reduce chance of unwanted errors like this.